pkg/apis/pipeline/pod/template.go

```diff
@@ -132,6 +132,18 @@ type Template struct {
 	// +optional
 	HostNetwork bool `json:"hostNetwork,omitempty"`
 
+	// HostUsers indicates whether the pod will use the host's user namespace.
+	// Optional: Default to true.
+	// If set to true or not present, the pod will be run in the host user namespace, useful
+	// for when the pod needs a feature only available to the host user namespace, such as
+	// loading a kernel module with CAP_SYS_MODULE.
+	// When set to false, a new userns is created for the pod. Setting false is useful for
+	// mitigating container breakout vulnerabilities even allowing users to run their
+	// containers as root without actually having root privileges on the host.
+	// This field is alpha-level and is only honored by servers that enable the UserNamespacesSupport feature.
```

Comment:
Copying "more info" block from [the other PR](https://github.com/tektoncd/pipeline/pull/9190/changes#diff-80408270974a80dab86725051cbdd1578bb10c654906e99a2c91db2d75302c4eR343)
```suggestion
                        This field is alpha-level and is only honored by servers that enable the UserNamespacesSupport feature.
                        More info: https://kubernetes.io/docs/tasks/configure-pod-container/user-namespaces
```